CRM/Admin/Form/MessageTemplates.php

```diff
@@ -169,7 +168,9 @@ public function buildQuickForm() {
     $this->applyFilter('__ALL__', 'trim');
     $this->add('text', 'msg_title', ts('Message Title'), CRM_Core_DAO::getAttribute('CRM_Core_DAO_MessageTemplate', 'msg_title'), TRUE);
 
-    $this->add('select', 'file_type', ts('File Type'), CRM_Core_SelectValues::documentFormat());
+    $options = array(ts(' Compose On-screen'), ts(' Upload Document'));
```

Comment:
Just for clarity I added a space between radio element and its label as its not possible via tpl. Anyways will trim that.